paddle/framework/operator.h

```diff
@@ -83,7 +83,7 @@ class OperatorBase {
     return boost::get<T>(attrs_.at(name));
   }
 
-  virtual std::string DebugString() const;
+  virtual void DebugPrint(std::ostream* os) const;
```

Comment:
这里可以 overload << 呀。根据 https://google.github.io/styleguide/cppguide.html#Reference_Arguments

> Don't go out of your way to avoid defining operator overloads. For example, prefer to define ==, =, and <<, rather than Equals(), CopyFrom(), and PrintTo(). 

这里的 DebugString 实际上就是应该避免定义的 PrintTo。